README.md

```diff
@@ -65,6 +65,8 @@ spec:
               value: "30"
             - name: VERBOSE # Optional, will log kube and cloudwatch statistics
               value: "false"
+            - name: LOG_LEVEL # Optional, defaults to "INFO". Allowable values: "DEBUG", "INFO", "ERROR", "OFF"
```

Comment:
should we remove the VERBOSE flag, since it is superseded by DEBUG level?

Reply:
It would be cleaner, but I was attempting to keep backward compat. I have no objection to dropping it.

Reply:
Done. It's pretty easy to set a log level.